src/ValueFormatters/TimeFormatter.php

```diff
@@ -58,17 +53,12 @@ public function format( $value ) {
 				. 'instances of DataValues\TimeValue' );
 		}
 
-		$formatted = $value->getTime();
-
 		$isoFormatter = $this->getOption( self::OPT_TIME_ISO_FORMATTER );
 		if ( $isoFormatter instanceof ValueFormatter ) {
-			$formatted = $isoFormatter->format( $value );
+			return $isoFormatter->format( $value );
 		}
```

Comment:
That's what https://gerrit.wikimedia.org/r/#/c/195883/ is about. Note that I reworked this patch completely, with that the class is useful again.